docs/contributing.md

```diff
@@ -30,6 +30,8 @@ Next, ensure that Composer accepts unstable releases from your local copies by c
 
 Finally, run `composer install` to complete the installation from the path repositories.
 
+After your local installation is set up, make sure you've enabled `debug` mode in **config.php**, and set `display_errors` to `On` in your php config. This will allow you to see error details for both Flarum and PHP. Furthermore, **the js will not be recompiled without running `php flarum cache:clear` if debug mode is off**.
```

Comment:
I'd use some sort of positive sentence here, like

```suggestion
After your local installation is set up, make sure you've enabled `debug` mode in **config.php**, and set `display_errors` to `On` in your php config. This will allow you to see error details for both Flarum and PHP. Debug mode also forces a re-compilation of Flarum's asset files on each request, removing the need to call `php flarum cache:clear` after each change to the extension's javascript or CSS.
```

The phrasing also needs to make a clear distinction to Flarum's compiled javascript ("assets") and the extension's compiled javascript (transpiled) described below. I think using the word "assets" for Flarum's compiled files makes sense. Avoiding mention to javascript should help prevent any confusion with the transpilation described lower in the page.